Cargo.toml

```diff
@@ -1,4 +1,11 @@
 [workspace]
 members = [
-  "sdk/*"
+  "sdk/core",
+  "sdk/cosmos",
+  # "sdk/event_grid",
+  "sdk/identity",
+  "sdk/iothub",
+  "sdk/key_vault",
+  # "sdk/service_bus",
```

Comment:
event_grid & service_bus currently require hyper. They compile individually, but not when the full workspace is compiled because they require `hyper_enabled`.